<launch>
    <arg name="path" default="/media/popular/T7/23_DataCon/4. 자율주행 차량 위치 추정/자율주행차량 로깅 데이터/" />
    <!-- <arg name="path" default="/home/popular/bagfile" /> -->
    <arg name="file_1" default="bagfile1" />
    <arg name="file_2" default="bagfile2" />
    <arg name="file_3" default="bagfile3" />
    <node pkg="rosbag" type="play" name="player" output="screen" 
    args="--clock $(arg path)$(arg file_1).bag $(arg path)$(arg file_2).bag $(arg path)$(arg file_3).bag"/>

</launch>